web/assets/css/transmission-app.scss

```diff
@@ -409,8 +409,8 @@ a {
     align-items: inherit;
     flex-direction: inherit;
 
-    &:not(:nth-child(1 of #mainwin-statusbar .speed-container)) {
-      width: 100px;
+    + .speed-container {
```

Comment:
Oooh nifty!

Reply:
Originally in search of `document.querySelector('.className')` from javascript side to pick only the 2nd occurrence and I ran across this cool little CSS selector. Naturally I've had to test this in normal CSS and it's doing as expected, that's something I wish I knew!